docs/build/guides/tokens/custom-sac-admin.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Set a custom admin account for a Stellar Asset Contract (SAC)
+hide_table_of_contents: true
+description: Set a custom administrator account on a deployed SAC
+---
+
+The [Stellar Asset Contract (SAC)](../../../tokens/stellar-asset-contract.mdx) includes functionality to set a custom administration account on the contract itself. This can be seen as a security precaution, similar to locking an asset's issuer account after creating the total supply of tokens. It should also be noted that an asset's issuer account can have a distinct set of signatures and weights, and these are unrelated to the SAC admin address.
+
+## Considerations
+
+To effectively utilize the SAC admin functionality, you should be aware of a few things first:
+
+- When a SAC is initially enabled for an issued asset, the `Admin` address on the contract defaults to the issuer account.
+- A SAC `Admin` address can be either a regular Stellar account (`G...`), or it can be a smart contract (`C...`) address. This opens up the possibility of programmatically minting and taking other administrative actions for an asset by way of a smart contract invocation.
+- Changing a SAC `Admin` address (initially) requires authorization from the asset's issuer account. After the `Admin` address has been set the first time, subsequent changes will require authorization from the _current_ `Admin` address.
```

Comment:
Instead of specifying the exceptions, we can just say that the admin change must be authorized by the current admin of the token. When token is first instantiated, its admin is set to its issuer.

Reply:
I did want to be a little more verbose in regard to this, though. When a dev is first considering a custom SAC admin, I think it's a bit counter-intuitive about having an issuer account that can mint/clawback, as well as a totally unrelated SAC admin that can do the same things. So, spending a bit more time and re-iterating in a couple ways seemed like a good approach to me.

That said, these points can be cleaned up and refined, for sure!

Reply:
On second thought, I may have misunderstood what you were getting at hehe. Yes, line 15 was fairly messy, and a much cleaner version will be in the upcoming PR